GOVERNANCE.md

```diff
@@ -0,0 +1,111 @@
+# Keycloak Governance
+
+* [Vision](#vision) 
+* [Maintainers](#maintainers) 
+* [Contributing](#contributing)
+
+## Vision
+
+Keycloak aims to be easy to use and lightweight. The project was founded to make it easy for application developers 
+to secure modern applications and services.
+
+The 80/20 rule is a core part of the vision behind Keycloak. Stating that 80% of requirements come from roughly 20%
```

Comment:
The second sentence does not have a subject.  If you drop the "Stating that," it works.

Reply:
Combined to a single sentence "The 80/20 rule, that states 80% of requirements come from around 20% of use cases, is a core part of the vision behind
Keycloak. "